backend/src/main/kotlin/org/loculus/backend/config/SecurityConfig.kt

```diff
@@ -4,7 +4,6 @@ import jakarta.servlet.http.HttpServletRequest
 import jakarta.servlet.http.HttpServletResponse
 import mu.KotlinLogging
 import org.loculus.backend.auth.Roles.EXTERNAL_METADATA_UPDATER
-import org.loculus.backend.auth.Roles.GET_RELEASED_DATA
```

Comment:
The `GET_RELEASED_DATA` role can probably deleleted, too?

Reply:
Done